_dockerPublisher/ReadMe_automaticPublishing.txt

```diff
@@ -0,0 +1,15 @@
+Automatic publishing of optaplanner-website
+===========================================
```

Comment:
No .adoc file?
Get used to the adoc file format and you'll never go back :)
  http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/